watertap/unit_models/zero_order/tests/test_brine_concentrator_zo.py

```diff
@@ -367,7 +367,6 @@ def test_no_tds_in_solute_list_error():
 db = Database()
 params = db._get_technology("brine_concentrator")
 
-
```

Comment:
Why is this change happening? According to PEP-8, I believe there should be two line spacing here.

Reply:
Oh, I am sorry that I accidently delete this blank line.